packages/nodes-base/nodes/Gllue/Gllue.node.ts

```diff
@@ -127,6 +127,14 @@ export class Gllue implements INodeType {
 				);
 				const body = {ids: contractIds, count: contractIds.length};
 				responseData = await getResponseByUri(uriGenerated, this.helpers.request, 'POST', body);
+			} else if (operation === 'add') {
+				const contractIds = this.getInputData().map(
+					(item, index) => this.getNodeParameter('id', index),
+				);
```

Comment:
maybe we could extract this code  before `if (operation == 'delete') {` since its repeated

Reply:
make sense